vendor/github.com/coreos/ignition/v2/config/merge/merge.go

```diff
@@ -240,10 +240,6 @@ func mergeStruct(parent reflect.Value, parentPath path.ContextPath, child reflec
 			resultField.Set(mergeStruct(parentField.Elem(), parentFieldPath, childField.Elem(), childFieldPath, resultFieldPath, transcript).Addr())
 			transcribeOne(parentFieldPath, resultFieldPath, transcript)
 			transcribeOne(childFieldPath, resultFieldPath, transcript)
-		case kind == reflect.Ptr && childField.IsNil() && fieldMeta.Name == "Compression":
-			// if a child's Compression field is nil, since it may have uncompressed
-			// Source that gets copied to result, Compression should not be set in
-			// result even if it's set in parent
```

Comment:
@cgwalters cherry picked a patch that never got merged, but I don't think we want to drop this yet

Reply:
Yeah that wasn't so much an opinionated exclusion as it was "include everything that go deps changed and figure it out later". :smile: